src/aequitas_cli/utils/configs_loader.py

```diff
@@ -8,10 +8,14 @@
 
 class Configs(object):
     def __init__(self, ref_groups_method='majority', fairness_threshold=0.8,
-                 attr_cols=None, report=True, score_thresholds=None, project_description={'title': '', 'goal': ''},
-                 ref_groups=None, db=None, fairness_measures=['Statistical Parity', 'Impact Parity', 'FDR Parity',
-                                                              'FPR Parity', 'FNR Parity', 'FOR Parity', 'TPR Parity',
-                                                              'Precision Parity']):
+                 attr_cols=None, report=True, score_thresholds=None,
+                 project_description={'title': '', 'goal': ''},
+                 ref_groups=None, db=None, fairness_measures=[
+                'Statistical Parity', 'Impact Parity', 'FDR Parity',
+                'FPR Parity', 'FNR Parity', 'FOR Parity', 'TPR Parity',
+                'Precision Parity'],
+                 plot_bias_disparities=[], plot_bias_metrics=[]):
```

Comment:
you never want to put mutable structures into signatures -- they can change and as a side effect change future invocations.

assuming these can't or shouldn't be derived from the `plotting` module, then in a similar manner they can be defined in this module or in this class -- but, with immutable types, (namely the `tuple`):

    def __init__(self, ..., fairness_measures=(...), ...):

or:

    all_fairness_measures = (...)

    def __init__(self, ..., fairness_measures=all_faireness_measures, ...):

etc.